modules/kml/src/gpx-loader.js

```diff
@@ -0,0 +1,47 @@
+/* global TextDecoder, DOMParser */
+import {geojsonToBinary} from '@loaders.gl/gis';
+import {gpx} from '@tmcw/togeojson';
+
+/** @typedef {import('@loaders.gl/loader-utils').LoaderObject} LoaderObject */
+
+// __VERSION__ is injected by babel-plugin-version-inline
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+const GPX_HEADER = `\
+<?xml version="1.0" encoding="UTF-8"?>
+<gpx`;
+
+function testText(text) {
```

Comment:
Can the `test` field just be set to that header?

Reply:
Presumably the `tests` field: https://github.com/visgl/loaders.gl/blob/74087169b419b7c696330b42d6d3413147a87f37/modules/loader-utils/src/types.d.ts#L17

Reply:
I updated `testDataAgainstText`, since the `tests` key wasn't being checked for string loaders.

https://github.com/visgl/loaders.gl/pull/1019/files#diff-d5cead0187a9abb5f92bec115b3747d0cb6bd16008fff2ab691e31815218ef81